src/core/room-lifecycle-manager.ts

```diff
@@ -404,6 +405,8 @@ export class RoomLifecycleManager {
     try {
       await this._doChannelWindDown(contributor).catch(() => {
         // If in doing the wind down, we've entered failed state, then it's game over anyway
+        // TODO: Another PR, but in the even if we get a failed channel, we still need to do the wind down
```

Comment:
Is the other PR already in review, or still to be started? If it's the former, can you add it to the PR body please?

Reply:
Not yet started, but now added an issue link